yeat/tests/data/configs/paired.cfg

```diff
@@ -0,0 +1,39 @@
+{
+    "samples": {
+        "sample1": [
+            "yeat/tests/data/short_reads_1.fastq.gz",
+            "yeat/tests/data/short_reads_2.fastq.gz"
+        ],
+        "sample2": [
+            "yeat/tests/data/Animal_289_R1.fq.gz",
+            "yeat/tests/data/Animal_289_R2.fq.gz"
+        ]
+    },
+    "assemblers": [
```

Comment:
Example of a complex YEAT job assembling 4 times for 2 samples.

-2 jobs for spades
-1 job for megahit
-1 job for Unicycler